### ISSUE TITLE
How fast should we expect the sparse implementation to be?

### ISSUE THREAD

**Issue by dexgs** [dilevin/computer-graphics-mass-spring-systems#37]
I believe my sparse implementation is mostly working since it computes the vertex position on the more complicated scenes in under a second. This is also true of the skirt simulation, but it is noticeably slower than the others. Is this expected?

**Comment by milomg** [comment]
Skirt starts up in under half a second for me in sparse. The FPS looks similar to this https://github.com/dilevin/computer-graphics-mass-spring-systems/blob/master/images/skirt.gif (I'd estimate this is around 10fps).

If you've combined everything into a single matrix expression, try sprinkling .eval in a few places to see if that improves perf.

**Comment by wenzhi-guo** [comment]
I agree with the above, there might be slight difference in speed depending on what machine you use. Also make sure you compiled everything in release mode.
